examples/05_stable_diffusion/scripts/download_pipeline.py

```diff
@@ -21,21 +21,26 @@
 @click.option(
     "--model-name",
     default="stabilityai/stable-diffusion-2-1-base",
-    help="Pretrained Model name",
+    help="Pretrained Model name.",
 )
-@click.option("--token", default="", help="access token")
+@click.option(
+    "--token",
+    default="",
+    help="Valid values: Huggingface user access token, 'true' to use token "
+    "generated with 'huggingface-cli login' (stored in ~/.huggingface) "
+    "or empty string to not use access token (default).")
```

Comment:
There is a small linter issue, please follow line 24 to change this to
```
    "or empty string to not use access token (default).",
)
```

Reply:
fixed